README.md

```diff
@@ -3,7 +3,6 @@
 [![Build Status](https://travis-ci.org/google/crc32c.svg?branch=master)](https://travis-ci.org/google/crc32c)
 [![Build Status](https://ci.appveyor.com/api/projects/status/moiq7331pett4xuj/branch/master?svg=true)](https://ci.appveyor.com/project/pwnall/crc32c)
 
-**This is not an official Google product.**
 New file format authors should consider
```

Comment:
"New file format authors should consider" .... is a tad cryptic sentence.... Can you elaborate?

Reply:
@pombredanne Thank you for the feedback!

The note means that if you're designing your own file format, you should consider newer hash functions. If you're writing code that needs to read/write data in an existing format that incorporates CRC32C, so you can't choose the hash function, then this library is for you.